packages/codemods/src/codeMods/__tests__/ComponentToCompat/ComponentToCompat.test.ts

```diff
@@ -0,0 +1,84 @@
+import { Project } from 'ts-morph';
```

Comment:
Is there a special reason we're using `__tests__` instead of `tests` in the path?

Reply:
I liked keeping them separate this way, especially since a lot of them require mocks. I'm not sure if it's the right approach. For right now I think this is okay to checkin and we can rearrange if things prove too difficult in the future.

Reply:
It seems like both conventions in open source, though in this repo just `tests` seems more common (including in a few cases having a central `tests` folder directly under `src`). So it might be better to omit the underscores for consistency.